code/datums/mood_events/generic_negative_events.dm

```diff
@@ -313,6 +313,11 @@
 	description = "I need something to cover my head..."
 	mood_change = -3
 
+/datum/mood_event/bald_reminder
+	description = "I was reminded that I can't grow my hair back at all! This is awful!"
```

Comment:
The description might confuse a few people, being used for both baldness and shavedness (facial hair) interactions with mirrors.

Reply:
Nahh it's fine man. I don't think that's ever gonna come up ingame.